Chaincase.UI/Components/FeeModal.razor

```diff
@@ -0,0 +1,99 @@
+﻿@using WalletWasabi.Exceptions
+@using WalletWasabi.Helpers
+@inherits ReactiveComponentBase<FeeViewModel>
+
+<IonModal @bind-Visible="@Visible">
+    <ion-header>
+        <ion-toolbar>
+            <ion-title>Select Fee</ion-title>
+            <ion-buttons slot="start">
+                <IonButton @onclick="CloseModal">Back</IonButton>
+            </ion-buttons>
+        </ion-toolbar>
+    </ion-header>
+
+    <ion-content class="ion-padding">
+        <p>
+            Bitcoin miners prioritize transactions that pay them the highest fee.
+            Only a limited number can be mined every 10 minute block.
+            These network fees are typically the least expensive on weekends and holidays and most costly when there new public interest or price changes.
```

Comment:
Who's the target audience here? This might be too wordy for a screen title. It's good info, maybe belongs in a sub screen that explains our fee calculations.

Reply:
Target audience is someone who's interested enough in the control to click on it rather than use the default